src/hooks/v2v3/contractReader/HeldFeesOf.ts

```diff
@@ -0,0 +1,25 @@
+import { ProjectMetadataContext } from 'contexts/projectMetadataContext'
+import { V2V3ContractsContext } from 'contexts/v2v3/V2V3ContractsContext'
+import { useContext, useEffect, useState } from 'react'
+import { sumHeldFees } from 'utils/v2v3/math'
+
+export function useHeldFeesOf() {
```

Comment:
wdyt about migrating a `loading` state into our contracts, rather than having each ref control the load? For example,

move the loading boolean into this hook, and return:
```
{heldFees, loading}
```

Reply:
Nice, I like it. Thanks ser